backend/api/simulation.py

```
@@ -2843,8 +2932,163 @@ async def generate_stream(db_session: Session):
+                    # Move to next scene
+                    next_scene_index = orchestrator.state.current_scene_index + 1
+                    next_scene = orchestrator.scenario.get('scenes', [])[next_scene_index]
+                    next_scene_id = next_scene.get('id')
+                    print(f"[DEBUG] (Timeout) Moving to next scene: index={next_scene_index}, id={next_scene_id}, title={next_scene.get('title')}")
+                    
+                    # No timeout message needed - using loading screen approach
+                    
+                    # Update orchestrator state
+                    orchestrator.state.current_scene_index = next_scene_index
+                    orchestrator.state.turn_count = 0
+                    print(f"[DEBUG] TURN COUNT RESET TO 0 ON TIMEOUT PROGRESSION")
```

Ruff (0.14.1)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)